openshift_pipelines/understanding-openshift-pipelines.adoc

```diff
@@ -0,0 +1,27 @@
+[id="understanding-openshift-pipelines"]
+= Understanding OpenShift Pipelines
+include::modules/openshift-pipelines-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: understanding-openshift-pipelines
+toc::[]
+
+:FeatureName: OpenShift Pipelines
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+{op-title} is a cloud-native, continuous integration and continuous delivery (CI/CD) solution based on Kubernetes resources. It uses link:https://tekton.dev[Tekton] building blocks to automate deployments across multiple platforms by abstracting away the underlying implementation details. Tekton introduces a number of standard link:https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/[Custom Resource Definitions (CRDs)] for defining CI/CD pipelines that are portable across Kubernetes distributions.
+
+[id="op-key-features"]
+== Key features
+
+* {op-title} are designed for decentralized teams that work on microservice based architecture.
```

Comment:
I'd also add something along OpenShift Pipelines is a serverless CI/CD system that runs pipelines with all the dependencies they require in isolated containers.

Reply:
@siamaksade - Made changes, please verify. Thanks !!

Reply:
@siamaksade ^^